packages/react-native-bpk-component-navigation-bar/src/isIphoneX.js

```diff
@@ -23,8 +23,12 @@ export default (() => {
   if (Platform.OS === 'android') {
     return false;
   }
+  const windowDimensions = DeviceInfo.getConstants().Dimensions.window;
+  if (!windowDimensions) {
```

Comment:
Could you clarify the logic and what is happening here?

Reply:
The interface for this has changed (https://github.com/facebook/react-native/pull/24878/files) and `window` is an `iOS` only property, and it can be null here because this file is not platform specific. So we first need to check if it is present before we use it. The end result should be the same as before

Reply:
Ok sounds great 😄